3rdparty/mlperftiny/api/submitter_implemented.h

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 EEMBC and The MLPerf Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+This file reflects a modified version of th_lib from EEMBC. All wrapped libc
+methods from th_libc.h and all testharness methods from th_lib.h are here.
```

Comment:
do we need to change the copyright notice for this file?

Reply:
Oh, I assumed the submitter_implemented is the part we wrote! I noticed there is a .cc file and our code is there.

Reply:
Thirdparty should be fine. I changed the ones that are under apps